internal/consensus/common_test.go

```diff
@@ -750,7 +749,6 @@ func makeConsensusState(
 	nValidators int,
 	testName string,
 	tickerFunc func() TimeoutTicker,
-	appFunc func(t *testing.T, logger log.Logger) abci.Application,
```

Comment:
Nice, very happy to see this pattern go away.

Reply:
hah, I mean I kinda like it.